spec/components/search_spec.rb

```diff
@@ -1690,4 +1690,29 @@ def indexed_post(*args)
     end
   end
 
+  context "advanced filter" do
+    let!(:post0) { Fabricate(:post, raw: 'this is the first post about advanced filter with length more than 50 chars') }
```

Comment:
minor but perhaps we can group the following tests under `context 'plugin extensions'` so that we can use `fab!` to create the posts?